src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,23 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+        {post.user && <UserInfo user={post.user} />}
+      </p>
```

Comment:
Not sure it it is correct
If the's no user int the post, it will say just "posted by"

Maybe you should male the whole <p> conditionally renderable?

```suggestion
       {post.user &&  <p>
          {' Posted by  '}
          <UserInfo user={post.user} />
      </p>}
```